bedrock/firefox/templates/firefox/accounts-features.html

```diff
@@ -15,81 +15,62 @@
 
 {% block body_class %}firefox-accounts-features state-fxa-default{% endblock %}
 
-{% block experiments %}
-  {% if switch('accounts-features-copy-experiment', ['en-US']) %}
-    {{ js_bundle('firefox_accounts_features_experiment') }}
-  {% endif %}
-{% endblock %}
-
-{% if variation == 'a' %}
-  {% set bookmarks_title = "Bookmark a recipe at home, shop for ingredients from your phone." %}
-  {% set passwords_title = "Binge watching waits for no one. Save passwords to log in faster." %}
-  {% set send_title = "Save commute time by sending links instantly between your devices." %}
-  {% set utm_content = '/firefox/accounts/features/v-a' %}
-{% else %}
-  {% set bookmarks_title = "Automatically update bookmarks with a click" %}
-  {% set passwords_title = "Save passwords safely" %}
-  {% set send_title = "Send links instantly between devices" %}
-  {% set utm_content = '/firefox/accounts/features/v-b' %}
-{% endif %}
-
 {% set cta_url = 'https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org:fxa.features&utm_campaign=may2018&utm_content=/firefox/accounts/features/' %}
 
 {% block content %}
 <main>
     <header class="fab-head">
       <div class="content">
-        <h1 class="fab-head-title">{{_('Firefox Account')}}</h1>
-        <p class="fab-head-subtitle">{{_('One easy login.<br> Epic perks.')}}</p>
+        <h1 class="fab-head-title">{{_('Move from device to device easily with a Firefox Account.')}}</h1>
         <div class="fab-head-cta">
-          {{ fxa_create_account_button('primary cta', 'features-header', 'may2018', 'fxa.features', utm_content, 'account-features') }}
+          {{ fxa_create_account_button('primary cta', 'features-header', 'jul2018', 'fxa.features', _('Create an Account'), 'button-light', utm_content, 'account-features') }}
         </div>
       </div>
     </header>
 
-    <h2 class="fab-intro">
```

Comment:
Without this `<h2>` all the perk headings need to move up a level.

Reply:
Doh!